playwright-tests/tests/infrastructure/infrastructure.near.spec.js

```diff
@@ -10,9 +10,9 @@ test.describe("Wallet is connected", () => {
     const aboutHeaderLink = await page.getByRole("link", { name: "About" });
     await expect(aboutHeaderLink).toBeVisible();
     await aboutHeaderLink.click();
-    await expect(await page.locator(".content-container")).toContainText(
-      "Introduction"
-    );
+    await expect(
```

Comment:
Maybe also add some expectation about not seeing the scrollbar for longer content